packages/reports/src/reports/blocks/Block.jsx

```diff
@@ -211,7 +214,32 @@ function Block({id, modified, callbacks, inline}) {
 
   const executeButton = <Button style={{minHeight: 40}} onClick={() => onSave(true)}>Save &amp; Execute</Button>;
 
-  const components = {blockPreview, apiInput, textEditor, codeEditor, blockSettings, executeButton, simpleUI};
+  const isStatlike = ![BLOCK_TYPES.GENERATOR, BLOCK_TYPES.VIZ, BLOCK_TYPES.SELECTOR].includes(block.type);
+
+  /** This is the (non-code) GUI editor for blocks that is meant for most simple cases.
+   * If the block is a stat-like type, then use a rich text editor to render its forms.
+   * Otherwise, a block type will need to have its own tailored Simple UI logic.
+   */
+  const uiEditor = isStatlike
```

Comment:
As we discussed in our call, the RichTextEditor should be merged into the SimpleUI umbrella, as it is just a special case of simple. But this is obviously fine for this PR

Reply:
Yep, that is part of my task for my next PR.